src/main/java/frc/robot/subsystems/intake/IntakeRollerStuff.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystems.intake;
+
+import frc.robot.hardware.digitalinput.IDigitalInput;
+import frc.robot.hardware.motor.IMotor;
+import frc.robot.hardware.signal.InputSignal;
+
+public record IntakeRollerStuff(
+	String logPath,
+	String digitalInputLogPath,
+	IMotor motor,
+	InputSignal voltageSignal,
+	IDigitalInput digitalInput
+) {
+
+	//@formatter:off
```

Comment:
remove

Reply:
But I want to keep these spaces, and the formatter erases them...

Reply:
no need here, u can remove